frontend/components/pages/BuildPage/build-page.component.tsx

```diff
@@ -128,7 +128,7 @@ function BuildPage({ build }: IBuildPageProps): JSX.Element {
           <Stacker orientation="vertical" gutter={16}>
             {/* {(isAdmin || ownedByMe) && ( */}
             {ownedByMe && (
```

Comment:
Insead of `ownedByMe` you should check for the existence of the `edit` link and ideally also use the href instead of hard coding the url. I'm now noticing that the link is missing though. Will add it.

Reply:
👍 Fixing in next PR

Reply:
I think it's still missing, I'll make that change when `edit` is part of the links